control/controlclient/direct.go

```diff
@@ -806,6 +807,7 @@ func (c *Direct) sendMapRequest(ctx context.Context, maxPolls int, cb func(*netm
 			}
 			setControlAtomic(&controlUseDERPRoute, resp.Debug.DERPRoute)
 			setControlAtomic(&controlTrimWGConfig, resp.Debug.TrimWGConfig)
+			controlknobs.SetDisableUPnP(resp.Debug.DisableUPnP)
```

Comment:
we should also set this false if there is no resp.Debug.

that's a bug with the existing ones too probably.

move all 3 lines before line 801 and make them be:

     controlknobs.SetDisableUPnP(resp.Debug != nil && resp.Debug.DisableUPnP)